client/bio-commandline/src/test/kotlin/uk/ac/ebi/biostd/client/cli/services/SubmissionServiceMigrationTest.kt

```diff
@@ -109,6 +109,15 @@ class SubmissionServiceMigrationTest(
         every { sourceClient.getExtByAccNo("S-BSST1") } returns extSubmission
         every {
             sourceClient.getReferencedFiles("/submissions/extended/S-BSST1/fileList/test-file-list/files")
-        } returns ExtFileTable(NfsFile(referencedFile.name, referencedFile, emptyList()))
+        } returns ExtFileTable(
+            NfsFile(
+                referencedFile.name,
+                "filePath",
```

Comment:
I think it'd be more realistic if you used these fields for their actual purpose like
"file1.txt"
"my/folder/file1.txt"
"Files/my/folder/file1.txt"